src/file.ts

```diff
@@ -35,13 +35,15 @@ async function zipBuildResult(buildResult: BuildResult): Promise<void> {
 }
 
 async function moveBuildsToExportDirectory(buildResults: BuildResult[], moveArchived?: boolean): Promise<void> {
-  const fullExportPath = path.resolve(RELATIVE_EXPORT_PATH);
-  core.startGroup(`Moving exports to ${fullExportPath}`);
-
-  await io.mkdirP(fullExportPath);
-
   const promises: Promise<void>[] = [];
   for (const buildResult of buildResults) {
+    const fullExportPath = path.resolve(
+      USE_PRESET_EXPORT_PATH ? path.dirname(buildResult.preset.export_path) : RELATIVE_EXPORT_PATH,
+    );
+    core.startGroup(`Moving exports to ${fullExportPath}`);
```

Comment:
Just this minor thing, can you move this `core.startGroup` to outside the loop? The message can be changed to something like `'Moving exports'` instead. Also friendly reminder to `yarn build` again once that's been changed.